Documentation/ApiOverview/Backend/BackendModules/InitializeModuleTemplate.rst.txt

```diff
@@ -0,0 +1,32 @@
+..  Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets
+..  Extracted from T3docs\BlogExample\Controller\BackendController
+
+..  code-block:: php
+    :caption: Class T3docs\\BlogExample\\Controller\\BackendController
```

Comment:
Backslash has to be escaped in rst files?

Reply:
Backslashes are used for escaping and have to be escaped themselves. Exceptions: in code-blocks or inliner code definitions. But a caption is just text